doc/rados/operations/monitoring.rst

```diff
@@ -345,19 +345,31 @@ the following::
 
 	ceph df
 
-The **RAW STORAGE** section of the output provides an overview of the
-amount of storage that is managed by your cluster.
+The output of ``ceph df`` for a theoretical 250 Kb object written to the 'foo_replicated' pool looks like this::
+
+        GLOBAL:
```

Comment:
this doesn't look aligned in columns like the actual output would be

Reply:
I've altered this, but we'll have to go over it bit by bit.

Reply:
this output is a bit different in master as well - here's a recent sample:

```--- RAW STORAGE ---
CLASS  SIZE     AVAIL    USED     RAW USED  %RAW USED
ssd    202 GiB  200 GiB  2.0 GiB   2.0 GiB       1.00
TOTAL  202 GiB  200 GiB  2.0 GiB   2.0 GiB       1.00
 
--- POOLS ---
POOL                   ID  PGS  STORED   (DATA)   (OMAP)   OBJECTS  USED     (DATA)  (OMAP)   %USED  MAX AVAIL  QUOTA OBJECTS  QUOTA BYTES  DIRTY  USED COMPR  UNDER COMPR
device_health_metrics   1    1  242 KiB   15 KiB  227 KiB        4  251 KiB  24 KiB  227 KiB      0    297 GiB  N/A            N/A              4         0 B          0 B
cephfs.a.meta           2   32  6.8 KiB  6.8 KiB      0 B       22   96 KiB  96 KiB      0 B      0    297 GiB  N/A            N/A             22         0 B          0 B
cephfs.a.data           3   32      0 B      0 B      0 B        0      0 B     0 B      0 B      0     99 GiB  N/A            N/A              0         0 B          0 B
test                    4   32   22 MiB   22 MiB   50 KiB      248   19 MiB  19 MiB   50 KiB      0    297 GiB  N/A            N/A            248         0 B          0 B
```

Reply:
This correctly-formatted table has been added to the docs.